Kconfig.zephyr

```
@@ -486,6 +486,12 @@ config BUILD_OUTPUT_UF2_FAMILY_ID
 	  name string. If the SoC in use is known by UF2, the Family ID will
 	  be pre-filled with the known value.
 
+config BUILD_OUTPUT_UF2_IGNORE_FLASH_BASE
+	bool "Ignore configured flash base for UF2 offset calculation"
```

similar proposal here, phrase so this is default behavior that can be disabled:
```suggestion
config BUILD_OUTPUT_UF2_BASE_OFFSET
	bool "UF2 flash base adjusted"
	default y
	help
	   UF2 conversion will default be adjusted with the FLASH_BASE_ADDRESS value.
	   Disable this setting to use 0x0 as conversion address.
```